bin/epaa.py

```diff
@@ -1031,7 +1034,12 @@ def __main__():
         if version not in EpitopePredictorFactory.available_methods()[method]:
             raise ValueError("The specified version " + version + " for " + method + " is not supported by epytope.")
 
-    thresholds = {"syfpeithi":50, "mhcflurry":500, "mhcnuggets-class-1":500, "mhcnuggets-class-2":500, "netmhc":500, "netmhcpan":500, "netmhcii":500, "netmhciipan":500}
+    thresholds = {"syfpeithi":60, "mhcflurry":500, "mhcnuggets-class-1":500, "mhcnuggets-class-2":500, "netmhc":500, "netmhcpan":500, "netmhcii":500, "netmhciipan":500}
```

Comment:
Who defined this "best practice" for the `syfpeithi` threshold? We used to work with a threshold of 50% and we should only change default values here if absolutely necessary since this changes the (default) results.

Reply:
So back in the day the group did an assessment which syfpeithi threshold would be the best to pick and it ended up being at 60%. So basically Stefan Stefanovic and his research team decided the "best practice" cut-off in that sense

Reply:
We can also discuss that further

Reply:
Okay we will stick to threshold 50